doc/setup/install.rst

```diff
@@ -525,13 +533,27 @@ Dependencies
 ^^^^^^^^^^^^
 
 Odoo dependencies are listed in the `requirements.txt` file located at the root of the Odoo
-community directory. Most of them can be installed with **pip**.
+community directory. Most of them can be installed with **pip** altough some libraries require
+installing other system package too.
 
 .. tip:: It can be preferable to not mix python modules packages between different instances of Odoo
          or with your system. You can use virtualenv_ to create isolated Python environments.
 
-Navigate to the path of your Odoo Community installation (`YourOdooCommunityPath`) and run **pip**
-on the requirements file:
+First, make sure all system dependencies are met:
+
+.. code-block:: console
+
+    # apt install libpq-dev libldap2-dev libsasl2-dev
+    # apt install python3-setuptools python3-wheel
+
+Then, ensure wheel in available within pip:
+
+.. code-block:: console
+    $ pip3 install wheel
```

Comment:
do we use wheel for this install?

Reply:
Surprisingly I had to in order to install stuff on Mathieu's machine